tests/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ add_test(NAME tst_ncio  COMMAND tst_ncio)
 
 # Brian Curtis: MPI Test
 add_executable(tst_ncio_mpi tst_ncio_mpi.F90)
-target_link_libraries(tst_ncio_mpi PRIVATE ncio::ncio NetCDF::NetCDF_Fortran)
+target_link_libraries(tst_ncio_mpi PUBLIC ncio::ncio NetCDF::NetCDF_Fortran MPI::MPI_Fortran)
```

Comment:
This should be PRIVATE not PUBLIC.
You don't need to link with NetCDF and MPI. They are PUBLIC in `ncio::ncio`

Reply:
Also, remove linking with `NetCDF::NetCDF_Fortran` from line 13.

Reply:
Fixed this.